src/utils/dispatcher.h

```diff
@@ -0,0 +1,54 @@
+#ifndef DISPATCHER_H
+#define DISPATCHER_H
+#include <memory>
+#include <glibmm.h>
+
+namespace utils {
+    /**
+     * @brief The Dispatcher, executes given function on gtk main loop.
```

Comment:
There is no doxygen, but there is more in the old `https://github.com/KoKuToru/gTox/blob/master/Source/Tox/Toxmm.h`...

I am using these doxygen-tags because my IDE displays these messages..
They are also visual different then normal comments.

I see normal comments as "not important" but nice to have.

I also tag todos with `//TODO: `, my current IDE doesn't highlight them, but others might..
(also `//HACK:` `//XXX:` `//FIXME:` ..)

But this rewrite is not finish yet, I will add the most important comments before merging it.

In my experince the generate doxygen will be hardly useful, we aren't writing a library..

but the whole `toxmm2::` and `utils::` should be documented (with doxygen-tags).
Everything else will probably just have a introduction above the class..